.github/workflows/e2e.yaml

```diff
@@ -69,9 +74,14 @@ jobs:
           distribution: "temurin"
           java-version: ${{ matrix.java }}
           cache: "maven"
+      - name: Set up Docker Compose
+        uses: docker/setup-compose-action@v1
+        with:
+          version: latest
```

Comment:
what does that mean?

Reply:
The current latest version of Docker Compose is v2.40.3.

Reply:
nit: can we use a fixed version for reproducibility please?